### ISSUE TITLE
[a11y]: PageHead reports The scrollable element <h1> with non-interactive content is not tabbable

### ISSUE THREAD

**Issue by jonrober-80** [carbon-design-system/ibm-products#6526]
### Package

@carbon/ibm-products

### Browser

Chrome

### Operating System

Other

### Package version

v2.54.x

### React version

^18.3.1

### Automated testing tool and ruleset

https://www.npmjs.com/package/accessibility-checker

### Description

When running the `PageHeader` component in "Simple page header with page title" mode (as per storybook - https://ibm-products.carbondesignsystem.com/?path=/story/ibm-products-components-page-header-pageheader--with-title&globals=viewport:basic)

the accessibility checker tool (https://www.npmjs.com/package/accessibility-checker) returns the following error about the header not being tabbable. 

```
10:58:33  - Message: The scrollable element <h1> with non-interactive content is not tabbable
10:58:33    Level: violation
10:58:33    XPath: /html[1]/body[1]/div[1]/div[1]/div[3]/div[1]/div[1]/div[1]/div[1]/section[1]/div[1]/div[1]/div[1]/div[1]/h1[1]
10:58:33    Snippet: <h1 class="c4p--page-header__title">
10:58:33    Help: https://able.ibm.com/rules/archives/2024.06.17/doc/en-US/element_scrollable_tabbable.html#%7B%22message%22%3A%22The%20scrollable%20element%20%3Ch1%3E%20with%20non-interactive%20content%20is%20not%20tabbable%22%2C%22snippet%22%3A%22%3Ch1%20class%3D%5C%22c4p--page-header__title%5C%22%3E%22%2C%22value%22%3A%5B%22VIOLATION%22%2C%22FAIL%22%5D%2C%22reasonId%22%3A%22fail_scrollable%22%2C%22ruleId%22%3A%22element_scrollable_tabbable%22%2C%22msgArgs%22%3A%5B%22h1%22%5D%7D
```

### Reproduction/example

https://ibm-products.carbondesignsystem.com/?path=/story/ibm-products-components-page-header-pageheader--with-title&globals=viewport:basic

### Steps to reproduce

Run the https://www.npmjs.com/package/accessibility-checker as described in the README against a PageHeader containing just a title (like the storybook example)

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems

**Comment by amal-k-joy** [comment]
Hi @jonrober-80 ,
I tried, but couldn't  reproduce the issue in [this variant](https://ibm-products.carbondesignsystem.com/?path=/story/ibm-products-components-page-header-pageheader--with-title&globals=viewport:basic) . Could you share a stackblitz with minimum reproduction or a recording? Also in this scenario, title in not scrollable.
![Image](https://github.com/user-attachments/assets/4c21b0f9-69db-472f-b50e-259c2bfe300e)
